tests/CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ add_subdirectory(http/clients)
 add_subdirectory(server)
 add_subdirectory(libtest)
 add_subdirectory(unit)
+add_subdirectory(certs)
```

Comment:
Is this executing by default for the `all` target, or only when explicitly building `build-certs`? or `testdeps`?